### PR TITLE
[DOCFIX] Update hive docs

### PR DESCRIPTION
**Pull request by yupeng9**
@TeddyBear1314  I think to set Alluxio properties, we need to config the hadoop `core-site.xml`. That's because Hive only hosts the metadata, the actual querying is done through MapReduce, which will use Hadoop config. For example, `fs.alluxio.impl` is set in Hadoop config, but not Hive.

### CODE REVIEW

File: docs/en/Running-Hive-with-Alluxio.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ Hive's Hadoop jobs will store its intermediate results in Alluxio. Please follow
 
 ### Add additional Alluxio site properties to Hive
 
-If there are any Alluxio site properties you want to specify for Hive, add those to `hive-site.xml`. For example,
-change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
+If there are any Alluxio site properties you want to specify for Hive, add those to `core-site.xml` to Hadoop configuration
+ directory on each node. For example, change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
```
Comment (calvinjia) [inline docs/en/Running-Hive-with-Alluxio.md]: Does adding `alluxio-site.properties` to the classpath (ie. add to the Hadoop conf folder) not work?

**Comment by TeddyBear1314** [comment]
@yupeng9 yes, your idea is more reasonable. I was only focusing on Hive itself and didn't think this clearly.
these properties are used by Hadoop and are suitable to be set in the Hadoop **'core-site.xml'**.
Thanks for your commits on the Alluxio Hive doc, I totally agree with your changes!

File: docs/en/Running-Hive-with-Alluxio.md (hunk 1 above)
Comment (yupeng9) [inline docs/en/Running-Hive-with-Alluxio.md]: I don't think that would work. Why would Hadoop understand alluxio's site properties? I think adding configs to `core-xml` is more appropriate because the idea is to append Alluxio config to Hadoop's config, and `core-site` is the place to specify config properties.

Moreover, `alluxio-site.properties ` contain both server and client configs. We should only need to add client-side properties to hadoop.

Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: hive or hdfs doesn't load the alluxio site properties, but the alluxio client invoked by hive should be able to create a Property instance based on the site property file if it is placed in hive classpath. At least this is the expected behavior

Comment (yupeng9) [inline docs/en/Running-Hive-with-Alluxio.md]: Okay, which is the recommended way? I don't see we suggest using `alluxio-site.properties` in any other docs. So for consistency, I prefer the PR as is. And if we decide to recommend a different way, we should do it in all docs in another PR. What do you think?

Comment (calvinjia) [inline docs/en/Running-Hive-with-Alluxio.md]: That works for me, could you please open a ticket to track this?

Comment (yupeng9) [inline docs/en/Running-Hive-with-Alluxio.md]: tracked at [ALLUXIO-2721](https://alluxio.atlassian.net/browse/ALLUXIO-2721)

**Comment by yupeng9** [comment]
@calvinjia @apc999 Any more comments?
